react-view/src/pages/MultiWidgetPage/index.tsx

```
@@ -0,0 +1,85 @@
+    <div>
+      {data.response.slots.map((datum) => {
+        let widgetData;
+        switch (datum.type) {
+          case "PRODUCT_INFO":
+            widgetData = datum.data as ProductInfoWidgetDataModel;
+            Component = ProductInfo;
+            break; 
+          default:
+            Component = FallbackComponent
+        }
+        return <Component widgetData={widgetData} />;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)